app/views/admin/users/_actions.html.erb

```diff
@@ -11,7 +12,7 @@
         <% end %>
         <% unless @last_email_verification_date %>
           <%= form_with url: verify_email_ownership_admin_user_path(@user), local: true do |f| %>
-            <%= f.button "Verify email address", class: "c-btn w-100 align-left" %>
+            <%= f.button "Verify email address", class: "c-btn w-100 align-left", data: { "no-modal": "" } %>
```

Comment:
This could be handled differently, but for now, I have a data attribute so this can be used as part of the check when a button in the dropdown menu is clicked.